<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-10-26T18:49:44 IST" name="Suite" finished-at="2025-10-26T18:49:58 IST" duration-ms="14120">
    <groups>
    </groups>
    <test started-at="2025-10-26T18:49:44 IST" name="TestChrome" finished-at="2025-10-26T18:49:58 IST" duration-ms="14120">
      <class name="listener_ExtentReport.TestClass">
        <test-method is-config="true" signature="setup(java.lang.String,java.lang.String,org.testng.ITestContext)[pri:0, instance:listener_ExtentReport.TestClass@740cae06]" started-at="2025-10-26T18:49:44 IST" name="setup" finished-at="2025-10-26T18:49:57 IST" duration-ms="12108" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://www.geeksforgeeks.org/software-testing/automation-testing-software-testing/]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[org.testng.TestRunner@2a693f59]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testLogo()[pri:1, instance:listener_ExtentReport.TestClass@740cae06]" started-at="2025-10-26T18:49:57 IST" name="testLogo" finished-at="2025-10-26T18:49:57 IST" duration-ms="433" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogo -->
        <test-method signature="testURL()[pri:2, instance:listener_ExtentReport.TestClass@740cae06]" started-at="2025-10-26T18:49:57 IST" name="testURL" finished-at="2025-10-26T18:49:57 IST" duration-ms="28" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://www.geeksforgeeks.org/software-testing/automation-testing-software-] but found [https://www.geeksforgeeks.org/software-testing/automation-testing-software-testing/]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://www.geeksforgeeks.org/software-testing/automation-testing-software-] but found [https://www.geeksforgeeks.org/software-testing/automation-testing-software-testing/]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at listener_ExtentReport.TestClass.testURL(TestClass.java:56)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testURL -->
        <test-method signature="testSkip()[pri:3, instance:listener_ExtentReport.TestClass@740cae06]" started-at="2025-10-26T18:49:57 IST" name="testSkip" finished-at="2025-10-26T18:49:57 IST" depends-on-methods="listener_ExtentReport.TestClass.testURL" duration-ms="1" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method TestClass.testSkip()[pri:3, instance:listener_ExtentReport.TestClass@740cae06] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method TestClass.testSkip()[pri:3, instance:listener_ExtentReport.TestClass@740cae06] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSkip -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:listener_ExtentReport.TestClass@740cae06]" started-at="2025-10-26T18:49:57 IST" name="tearDown" finished-at="2025-10-26T18:49:58 IST" duration-ms="427" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- listener_ExtentReport.TestClass -->
    </test> <!-- TestChrome -->
  </suite> <!-- Suite -->
</testng-results>
